feature/src/main/java/org/woowatechcamp/banchan/feature/detail/ProductDetailViewModel.kt

```diff
@@ -166,4 +171,18 @@ class ProductDetailViewModel @Inject constructor(
             }
         }
     }
+
+    private fun getExistState() {
+        viewModelScope.launch {
+            isExistsInProgressUseCase()
+                .onSuccess {
+                    it.collectLatest { isExist ->
+                        _isExistInProgress.emit(isExist)
```

Comment:
여기도 value로 값을 전달해도 될 것 같습니다!
```suggestion
                        _isExistInProgress.value = isExist
```

Reply:
value로 값을 전달하면 `flowWithLifecycle에서` 값 발행이 안되서 일단 `emit`으로 작성해 두었습니다.
원인이 뭔질 모르겠네요

Reply:
흠... `StateFlow` 클래스 내부에서도 emit의 구현양식을 보면 value에 값을 집어넣고 있는데.. 흠.. 뭔가 문제가 있는걸까요..

<img width="318" alt="image" src="https://user-images.githubusercontent.com/70698151/186625318-e086dfee-29f3-4f1b-8d5c-9da83a06097e.png">

Reply:
제가 정신이 나가서 그런걸 수도 있으니 나중에 다시 확인해보겠습니다😭